app/profile/[id]/page.tsx

```diff
@@ -1,11 +1,12 @@
 'use client';
 
-import { LoadingArea, SettingIcon } from '@/components/atoms';
+import { LoadingArea } from '@/components/atoms';
 import {
   BackButton,
   GlobalNavigationBar,
   HeaderBar,
 } from '@/components/molecules';
+import { SettingButton } from '@/components/molecules/header-bar/header-bar-setting-button';
```

Comment:
```suggestion
import { SettingButton } from '@/components/molecules';
```

SettingButton 베럴파일에 추가해주세용!

Reply:
음마야 ㅠㅠ